docs/modules/user-guide/partials/proc_refering-to-a-parent-devfile-in-a-devfile.adoc

```diff
@@ -2,18 +2,22 @@
 = Refering to a parent devfile in a devfile
 
 [role="_abstract"]
-It is possible to refer to another devfile as a parent to a given devfile. If a parent is specified, the devfile will inherit all behaviour from the parent. Part of the content of the parent can be overridden in the child devfile. That allows refering a devfile (the parent) from a distinct devfile and makes it possible to reuse in multiple devfiles the same parent (the stack).
-
 .Prerequisites
 
 * xref:adding-schema-version-to-a-devfile.adoc[]
 * xref:adding-a-name-to-a-devfile.adoc[]
 
 
+This section teaches you how to specify a parent devfile. You can choose a devfile and refer to another devfile as its parent. If you specify a parent, the child devfile inherits all its behavior from its parent devfile. Still, you can use the child devfile to override certain content from the parent devfile. If you override the correct content, you can reuse the same parent devfile in multiple other devfiles. If you do reuse a parent devfile, the parent turns into a stack that is used in multiple other devfiles.
```

Comment:
I can't verify it now, but I am pretty sure that you can't say "teach" in our guides. I've never come across it and it doesn't seem like the tone of our guides.

Reply:
which content is the correct content to override?

Reply:
"If you do reuse a parent devfile, the parent turns into a stack that is used in multiple other devfiles."
is the parent devfile reused or turns into a stack? Both? Why does one time reuse automatically result in the devfile being used by multiple other devfiles?

Reply:
Good point. Another question I can't address now. It's probably something we can tackle in the separate issue for properly structure our procedures: https://github.com/devfile/api/issues/402